### PR TITLE
update brewfile with instructions

### PR DESCRIPTION
**Pull request by BackerKitEngineering**

### CODE REVIEW

**Comment by BackerKitEngineering** [comment]
@srt32  for your consideration sir! -J & L

File: Brewfile [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+# This is a brewfile. It is a manifest that homebrew can use to setup your dev environment. More on brew bundle here - https://github.com/Homebrew/homebrew-bundle
+
+# To execute:
+# cd/path-to-dotfiles-repo
+# brew bundle
+
+
+# Read more about these installations in our documentation:
+# https://github.com/BackerKit/BackerKit/docs/setup_dev_environment.md
+# https://github.com/BackerKit/BackerKit/docs/setup_dev_tools.md
```
Comment (srt32) [inline Brewfile:11]: 👍  on the content. Why put it here as opposed to in the `md` files?

**Comment by srt32** [comment]
👍

File: Brewfile (hunk 1 above)
Comment (johntrandall) [reply Brewfile:11]: I'm not sure this whole dotfiles repo is the way to go.... TBDiscussed where this stuff goes once Lindsey and I are done with the readme.

**Pull request merged**